<ui version="4.0" >
 <class>MessengerWindow</class>
 <widget class="QMainWindow" name="MessengerWindow" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>282</width>
    <height>534</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>RetroShare  Messenger</string>
  </property>
  <property name="windowIcon" >
   <iconset resource="images.qrc" >:/images/rstray3.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget" >
   <layout class="QGridLayout" >
    <property name="leftMargin" >
     <number>0</number>
    </property>
    <property name="topMargin" >
     <number>0</number>
    </property>
    <property name="rightMargin" >
     <number>0</number>
    </property>
    <property name="bottomMargin" >
     <number>0</number>
    </property>
    <property name="horizontalSpacing" >
     <number>0</number>
    </property>
    <property name="verticalSpacing" >
     <number>0</number>
    </property>
    <item row="0" column="0" >
     <widget class="QFrame" name="messengerframetop" >
      <property name="frameShape" >
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow" >
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout" >
       <property name="leftMargin" >
        <number>0</number>
       </property>
       <property name="topMargin" >
        <number>0</number>
       </property>
       <property name="rightMargin" >
        <number>0</number>
       </property>
       <property name="bottomMargin" >
        <number>0</number>
       </property>
       <item row="0" column="0" >
        <layout class="QGridLayout" >
         <property name="leftMargin" >
          <number>4</number>
         </property>
         <property name="topMargin" >
          <number>4</number>
         </property>
         <property name="rightMargin" >
          <number>4</number>
         </property>
         <property name="bottomMargin" >
          <number>4</number>
         </property>
         <property name="horizontalSpacing" >
          <number>6</number>
         </property>
         <property name="verticalSpacing" >
          <number>6</number>
         </property>
         <item row="0" column="0" >
          <widget class="QToolButton" name="avatarButton" >
           <property name="toolTip" >
            <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">&#xd;
p, li { white-space: pre-wrap; }&#xd;
&lt;/style>&lt;/head>&lt;body style=" font-family:'Sans Serif'; font-size:9pt; font-weight:400; font-style:normal; text-decoration:none;">&#xd;
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">Click to change your avatar&lt;/p>&lt;/body>&lt;/html></string>
           </property>
           <property name="styleSheet" >
            <string>border-image: url(:/images/mystatus_bg.png);</string>
           </property>
           <property name="icon" >
            <iconset resource="images.qrc" >:/images/retrosharelogo1.png</iconset>
           </property>
           <property name="iconSize" >
            <size>
             <width>70</width>
             <height>70</height>
            </size>
           </property>
           <property name="autoRaise" >
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="1" >
          <layout class="QGridLayout" >
           <property name="leftMargin" >
            <number>4</number>
           </property>
           <property name="topMargin" >
            <number>4</number>
           </property>
           <property name="rightMargin" >
            <number>4</number>
           </property>
           <property name="bottomMargin" >
            <number>4</number>
           </property>
           <property name="horizontalSpacing" >
            <number>6</number>
           </property>
           <property name="verticalSpacing" >
            <number>6</number>
           </property>
           <item row="0" column="0" >
            <widget class="QComboBox" name="statuscomboBox" >
             <property name="enabled" >
              <bool>false</bool>
             </property>
             <property name="toolTip" >
              <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'Arial'; font-size:8pt; font-weight:400; font-style:normal; text-decoration:none;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">Change your Status and your display&lt;/p>
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">picture here.&lt;/p>&lt;/body>&lt;/html></string>
             </property>
             <property name="insertPolicy" >
              <enum>QComboBox::NoInsert</enum>
             </property>
             <property name="frame" >
              <bool>true</bool>
             </property>
             <property name="modelColumn" >
              <number>0</number>
             </property>
             <item>
              <property name="text" >
               <string>Online</string>
              </property>
              <property name="icon" >
               <iconset resource="images.qrc" >:/images/donline.png</iconset>
              </property>
             </item>
             <item>
              <property name="text" >
               <string>Busy</string>
              </property>
              <property name="icon" >
               <iconset resource="images.qrc" >:/images/dbusy.png</iconset>
              </property>
             </item>
             <item>
              <property name="text" >
               <string>Be right Back</string>
              </property>
              <property name="icon" >
               <iconset resource="images.qrc" >:/images/dbrb.png</iconset>
              </property>
             </item>
             <item>
              <property name="text" >
               <string>Away</string>
              </property>
              <property name="icon" >
               <iconset resource="images.qrc" >:/images/daway.png</iconset>
              </property>
             </item>
             <item>
              <property name="text" >
               <string>In a Call</string>
              </property>
              <property name="icon" >
               <iconset resource="images.qrc" >:/images/dphone.png</iconset>
              </property>
             </item>
             <item>
              <property name="text" >
               <string>Out of lunch</string>
              </property>
              <property name="icon" >
               <iconset resource="images.qrc" >:/images/dlunch.png</iconset>
              </property>
             </item>
             <item>
              <property name="text" >
               <string>Apear Offline</string>
              </property>
              <property name="icon" >
               <iconset resource="images.qrc" >:/images/dhidden.png</iconset>
              </property>
             </item>
            </widget>
           </item>
           <item row="1" column="0" colspan="2" >
            <widget class="QComboBox" name="messagecomboBox" >
             <property name="toolTip" >
              <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'Arial'; font-size:8pt; font-weight:400; font-style:normal; text-decoration:none;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">Share a personal message with your contacts&lt;/p>&lt;/body>&lt;/html></string>
             </property>
             <property name="editable" >
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="0" >
     <layout class="QGridLayout" >
      <property name="leftMargin" >
       <number>9</number>
      </property>
      <property name="topMargin" >
       <number>9</number>
      </property>
      <property name="rightMargin" >
       <number>9</number>
      </property>
      <property name="bottomMargin" >
       <number>9</number>
      </property>
      <property name="horizontalSpacing" >
       <number>6</number>
      </property>
      <property name="verticalSpacing" >
       <number>6</number>
      </property>
      <item row="1" column="0" colspan="2" >
       <widget class="QTreeWidget" name="messengertreeWidget" >
        <property name="contextMenuPolicy" >
         <enum>Qt::CustomContextMenu</enum>
        </property>
        <property name="iconSize" >
         <size>
          <width>18</width>
          <height>18</height>
         </size>
        </property>
        <property name="columnCount" >
         <number>0</number>
        </property>
       </widget>
      </item>
      <item row="0" column="0" >
       <widget class="QLineEdit" name="searchlineEdit" />
      </item>
      <item row="0" column="1" >
       <widget class="QToolButton" name="addIMAccountButton" >
        <property name="toolTip" >
         <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'Arial'; font-size:8pt; font-weight:400; font-style:normal; text-decoration:none;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">Add a Friend&lt;/p>&lt;/body>&lt;/html></string>
        </property>
        <property name="text" >
         <string/>
        </property>
        <property name="icon" >
         <iconset resource="images.qrc" >:/images/dadd.png</iconset>
        </property>
        <property name="iconSize" >
         <size>
          <width>22</width>
          <height>22</height>
         </size>
        </property>
        <property name="toolButtonStyle" >
         <enum>Qt::ToolButtonTextBesideIcon</enum>
        </property>
        <property name="autoRaise" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="2" column="0" >
     <widget class="QFrame" name="logoframe" >
      <property name="frameShape" >
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow" >
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>282</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="images.qrc" />
 </resources>
 <connections/>
</ui>
